Add notes and handlers attribute in agent /events API example (#2657)
sensu/sensu-docs: content/sensu-go/5.21/reference/checks.md

### Spec attributes

{{% notice note %}}
**NOTE**: Spec attributes are not required when sending an HTTP `POST` request to the [agent](../agent/#events-post) or [backend](../../api/events/#create-a-new-event) /events API.

> In that case, the spec attributes are listed as individual [top-level attributes](#top-level-attributes) instead, and there is no `spec` object in the check definition.

Reviewer: I'm not sure of this wording.
```suggestion
When doing so, the spec attributes are listed as individual [top-level attributes](#top-level-attributes) instead, and there is no `spec` object in the check definition.
```

> When doing so, the spec attributes are listed as individual [top-level attributes](#top-level-attributes) instead, and there is no `spec` object in the check definition.

Author: ```suggestion
When doing so, the spec attributes are listed as individual [top-level attributes](#top-level-attributes) in the check definition instead.
```
